dockstore-common/src/main/java/io/dockstore/common/DescriptorLanguage.java

```diff
@@ -84,9 +96,10 @@ public static DescriptorLanguage convertShortStringToEnum(String descriptor) {
             return WDL;
         case NFL_STRING:
             return NEXTFLOW;
+        case "service":
```

Comment:
Is there supposed to be a SERVICE_STRING somewhere?

Reply:
Rats, I'll clean this up in the next pass.